### PR TITLE
Simplify deblend field

### LINKED ISSUE #60: deblend_field function is too complex
need to separate the individual steps further.

### PR DESCRIPTION
**Pull request by b-biswas**
Here I am trying to simplify an break down the deblend_field function into sub functions. 
I also defined a class for IdentifyNorm to so that I don't have to repeat `if normalizer is None:` condition everywhere
Fixes #60

### CODE REVIEW

File: debvader/deblender.py [hunk 1]
```diff
@@ -291,6 +304,37 @@ def get_deblending_meta_data(self, res_deblend=None):
...
+            epistemic_uncertainty = deblend(
+                np.std(
+                    deblend(
+                        self.net,
+                        np.array([cutout_image] * 100),
+                        normalizer=self.normalizer,
+                    )[0],
+                    axis=0,
+                )
+            )
```
Comment (aboucaud) [inline debvader/deblender.py:325]: Could you add a little comment in the docstring about what computation is performed ?
I'm a bit confused by the `deblend(std(deblend(...)))`

My guess is that this is a mistake and the first `deblend` is actually a `list`.

**Review by aboucaud — COMMENTED** [review]
Appart from one thing to check, I really like the simplification you came with.
The identity normalizer is nice.

File: debvader/deblender.py (hunk 1 above)
Comment (b-biswas) [reply debvader/deblender.py:325]: I agree. This is somewhere I wanted to ask Bastien what he was trying to do
In my opinion... He was probably just trying to feed cutouts to the deblender 100 times to see the epistemic uncertainty but I am not sure

Comment (b-biswas) [reply debvader/deblender.py]: I think just removing the first deblend would solve the issue here. 
Then np.std is calculating the pixel-wise std over the predictions.

I just push a commit (after testing that the function works)

**Pull request merged**